conf/application.conf

```diff
@@ -23,11 +23,10 @@ entityfacts {
 }
 
 index {
-	home: "."
-	name: "authorities"
+	hosts: ["localhost"] # ["weywot3.hbz-nrw.de", "weywot4.hbz-nrw.de", "weywot5.hbz-nrw.de"]
```

Comment:
localhost seems to be potentially harmful - depending on what server you start the program and if this is really taken into acount?

Reply:
ah, ok . I also overlooked the '#' . So +1